src/wallet.js

```diff
@@ -1000,12 +1005,12 @@ const wallet = {
     tokens.clearDepositPercentage();
     this.clearRewardLockConstant();
     this.clearNetworkBestChainHeight();
-    storage.removeItem('wallet:started');
-    storage.removeItem('wallet:backup');
-    storage.removeItem('wallet:locked');
-    storage.removeItem('wallet:tokens');
-    storage.removeItem('wallet:sentry');
-    storage.removeItem('wallet:type');
+    StorageProxy.getStorage().removeItem('wallet:started');
```

Comment:
I think we should create a `const storage = StorageProxy.getStorage()` to avoid duplicate code.

Reply:
Done